### DISCUSSION TITLE
Is there any official or encouraged EU cookie law handling for orchard core?

### DISCUSSION THREAD

**Discussion by bashuss** [OrchardCMS/OrchardCore#9764] [Q&A]
Just needing to get my first orchard core page running according to EU regulations.
So I am wondering about cookies.

Will there be an official way to handle this, as it may have impact on some existing or upcoming services (integrated analytics, Recapture?, google fonts, youtube, advertizing) ...

For now, I can keep it really simple, but I have seen in shop systems, that make this property part of the user profile as an extended custom property.

So which way did you choose implementing this?
Is there a plan to support this in an official way in orchard core?

best regards,
Bernd

**Comment by Piedone** [comment]
Check this out: https://github.com/OrchardCMS/OrchardCore/issues/3580

**Reply by hishamco** [reply]
It's there in [Orchard Core Contrib](https://github.com/OrchardCoreContrib/OrchardCoreContrib.Modules/tree/dev/src/OrchardCoreContrib.Gdpr)

**Comment by bashuss** [comment]
Thanks for your answers!
I'll check it out
